novawallet/Modules/Vote/Governance/TinderGov/TinderGovViewModelFactory.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+protocol TinderGovViewModelFactoryProtocol {
+    func createTinderGovReferendumsSection(
+        for referendums: [ReferendumLocal]?,
+        locale: Locale
+    ) -> ReferendumsSection?
+}
+
+struct TinderGovViewModelFactory: TinderGovViewModelFactoryProtocol {
+    func createTinderGovReferendumsSection(
+        for referendums: [ReferendumLocal]?,
```

Comment:
Did you add votes in the next PR as we discussed?

Reply:
yep, I did